src/std/text.ab

```diff
@@ -1,14 +1,36 @@
-/// Replaces all occurences of a pattern in the content with the provided replace text.
+// Replace this with builtin if/when we get around to writing one.
+#[allow_absurd_cast]
+fun bash_version(): Num {
+    let major = trust $ echo "\$\{BASH_VERSINFO[0]}" $ as Num
+    let minor = trust $ echo "\$\{BASH_VERSINFO[1]}" $ as Num
+    return (major * 100) + minor
+}
+
+/// Replaces all occurrences of a pattern in the content with the provided replace text.
 pub fun replace(source, search, replace) {
-    return "\$\{source//{search}/{replace}}"
+    // Here we use commands for assigning values to avoid the side effect of Amber
+    // Escape `\` as `\\`.
+    trust $ {nameof search}="\$\{{nameof search}//\\\\/\\\\\\\}" $
+    if bash_version() >= 502 {
+        trust $ {nameof replace}="\$\{{nameof replace}//\\\\/\\\\\\\\}" $
+    } else {
+        trust $ {nameof replace}="\$\{{nameof replace}//\\\\/\\\\}" $
```

Comment:
I still don't understand why you're doing this, especially as you're not doing so in `replace_one`; surely this will be a no-op?  Suggest removing this line, and also adding a blank line before the `return`.

Reply:
I missed it. Thank you!